src/features/projectsV2/ProjectsMap/MapFeatureExplorer/MapFeatureExplorer.module.scss

```diff
@@ -9,26 +9,117 @@
 
 .exploreMainContainer {
   max-width: 182px;
-  background-color: $backgroundColor;
+  background-color: #fff;
   border-radius: 12px;
   margin-top: 10px;
-  padding: 14px 16px;
+  padding: 12px;
   box-shadow: 0px 6px 10px rgba(0, 0, 0, 0.12);
+  max-height: 485px;
+
+  &::-webkit-scrollbar {
+    display: none;
+  }
 
   @include xsPhoneView {
     position: absolute;
+    max-height: max-content;
+    min-width: 293px;
+    border-radius: unset;
+    right: 0px;
+    top: -10px;
```

Comment:
Why do this instead of making margin-top = 0px? This makes it tougher to understand how the element is positioned.